### ISSUE TITLE
Error Message on load

### ISSUE THREAD

**Issue by ctbritt** [mouse0270/module-credits#23]
Heya-- getting this error message on world load: 
```
preview.mjs:10 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'toUpperCase')
[Detected 1 package: module-credits]
    at preview.mjs:10:22
    at Array.sort (<anonymous>)
    at new PreviewDialog (preview.mjs:9:44)
    at module.mjs:167:6
(anonymous) @ preview.mjs:10
PreviewDialog @ preview.mjs:9
(anonymous) @ module.mjs:167
```
Doesn't seem to affect anything, so this is more of an FYI than anything.

**Comment by mouse0270** [comment]
A module you have installed does not have a title properly setup is my best guess.

Could you provide me a list of modules?

**Comment by ctbritt** [comment]
i ran it with only your module enabled and got the same message.

**Comment by mouse0270** [comment]
Well, it's my module reads every module installed, regardless of if they are active or not to find their readmes and changelog files.

**Comment by mouse0270** [comment]
No update - Assuming issue was fixed with v1.1.8
